manticore/platforms/linux.py

```diff
@@ -65,10 +65,7 @@ def __init__(self, path, flags):
         self.file = open(path, mode)
 
     def __getstate__(self):
-        state = {}
-        state['name'] = self.name
-        state['mode'] = self.mode
-        state['closed'] = self.closed
```

Comment:
this gets rid of saving 'closed'?

Reply:
also, one reason we were doing it the previous way is so it's pretty easy to just copy/paste a line when we want to add attributes to this class. i do think the new style of formatting is better though..maybe we can compromise, a la

```
state = {
  'name': self.name,
  'mode': self.mode,
}
```

Reply:
the closed property was just introduced a few days ago, i just merged master back in and fixed the conflict